cmd/monitoring/README.md

```diff
@@ -0,0 +1,75 @@
+# Solana Onchain monitoring
+
+
+## Run locally
+
+The monitor requires a schema registry, a kafka instance and a zookeper.
+You can use the docker-compose.yml file in `smartcontractkit/gokit` to start these up.
```

Comment:
`smartcontractkit/gokit` is still internal, so perhaps we could copy the `docker-compose.yml` over here under `/ops`, or add an option (implement requirements) to our Pulumi local env, or just make the `gokit` public.